python/ray/autoscaler/aws/example-minimal.yaml

```diff
@@ -14,4 +10,3 @@ provider:
 # How Ray will authenticate with newly launched nodes.
 auth:
     ssh_user: ubuntu
```

Comment:
This line wasn't in the hardcoded "minimal" yaml from getting-started. I have a feeling it's necessary so I left it in for now, but I'm not actually sure.